infrastructure/staging/backup.tf

```diff
@@ -0,0 +1,71 @@
+resource "aws_backup_plan" "workflow-default" {
+  name = "workflow-default"
```

Comment:
We might want to leave out the AWS backup stuff in the staging system completely. These resources might collide with the production ones actually.

Reply:
removed the backup from staging